platforms/core-configuration/base-diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/internal/TextReportRenderer.java

```diff
@@ -67,13 +66,7 @@ public void startProject(ProjectDetails project) {
     }
 
     protected String createHeader(ProjectDetails project) {
-        String header = StringUtils.capitalize(project.getDisplayName());
-
-        String description = project.getDescription();
-        if (GUtil.isTrue(description)) {
-            header = header + " - " + description;
-        }
-        return header;
+        return StringUtils.capitalize(project.getDisplayName());
```

Comment:
Printing the description here led to really ugly formatting for multi-line descriptions